### DISCUSSION TITLE
requireNativeComponent: "RNSScreenStackHeaderConfig" was not found in the UIManager when running android app

### DISCUSSION THREAD

**Discussion by gunayakay** [software-mansion/react-native-screens#1486] [Q&A] [answered]
I am getting such an error in android emulator. I reset my metro server, created a new project, reinstalled dependies, but my problem was not solved. Is there anyone who can help
![erorr](https://user-images.githubusercontent.com/101050062/175034993-25739661-8517-4959-82e5-32d9d5f57865.png)
?

**Comment by tboba** [comment, marked as answer]
Hi @gunayakay, thanks for submitting this problem!
Do you know if this problem still occurs for you in react-native-screens 3.26.0? If so, let's move to the issue, related to that problem: https://github.com/software-mansion/react-native-screens/issues/1614.

Thanks!
